lib/connect.js

```diff
@@ -1,345 +1,125 @@
-const { EventEmitter } = require('events');
-const fs = require('fs');
-const async = require('async');
-const {
-  includes,
-  clone,
-  assign,
-  isString,
-  isFunction,
-  omit
-} = require('lodash');
 const { MongoClient } = require('mongodb');
-const { parseConnectionString } = require('mongodb/lib/core');
-const Connection = require('./extended-model');
 const createSSHTunnel = require('./ssh-tunnel');
+const Connection = require('./extended-model');
 
 const debug = require('debug')('mongodb-connection-model:connect');
 
-const needToLoadSSLFiles = (model) =>
-  !includes(['NONE', 'UNVALIDATED'], model.sslType);
```

Comment:
`sslType` doesn't seem to exist on connection-model, so this false condition would never occur. Probably a bug. I think we can just avoid it.